float32 yaw_input
float32 yaw_setpoint
float32 distance_input
float32 distance_setpoint
